charts/chainlink-cluster/values.yaml

```
@@ -361,6 +361,16 @@ ingress:
                 name: mockserver
                 port:
                   number: 1080
+    - host: chainlink-grafana.local
+      http:
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+            backend:
+              service:
+                name: grafana
```

```suggestion
                name: app-grafana
```

Shouldn't this be the same as the devspace value?

---

No I dontthink so as it depends on release-name here and its not present for other services targets